docs/tutorial/index.md

```diff
@@ -2,16 +2,16 @@
 title: Gatsby.js Tutorial
 ---
 
-We’re so happy you decided to try using Gatsby. This tutorial has an initial introduction plus seven more parts that'll walk you from getting started developing and building Gatsby sites to deploying a finished and polished high performance static PWA.
+Welcome to Gatsby! We’re glad you’re here. The goal of this tutorial is to guide you through setting up and deploying your first Gatsby site using a starter template. As we walk through that process, we’ll introduce some more general web development topics, and go over the underlying structure of a Gatsby site. 
 
-Note: If you prefer a quick start rather than step-by-step tutorial, see the [getting started page](/docs/).
+The full tutorial is intended to be as accessible as possible to people without much web development experience (yet!) — no need to be an expert. If you prefer to jump straight to code, feel free to skip the step-by-step tutorial and see the [quick start](/docs/) page.
 
-This tutorial is for _everyone_! You do not need to be a programmer or React.js expert. We'll walk you through things.
+## Upcoming sections
 
-0.  [Programming basics](/tutorial/part-zero/): If you are new to programming, go here for an introduction to some basics like terminal, code editors, and browser consoles.
-1.  [Introduction to Gatsby basics](/tutorial/part-one/): Starting new projects, developing, and deploying sites.
-1.  [Introduction to using CSS in Gatsby](/tutorial/part-two/): Explore libraries like Typography.js and CSS Modules.
-1.  [Building nested layouts in Gatsby](/tutorial/part-three/): Layouts are sections of your site that are reused across multiple pages like headers and footers.
+0. [**Set up your development environment**](/tutorial/part-zero/): We’ll start by making sure you have installed all the tools you’ll need to begin development on a Gatsby site, and generate our very first site.
+1. [**Familiarize with Gatsby building blocks**](//tutorial/part-one/). We’ll tour the code we’ve generated for our new Gatsby site and get introduced to the Gatsby project code structure.
+2. [**Add styles to your site.**](/tutorial/part-two/) We’ll take a look at how we can start making changes to the presentational layer of the site by experimenting with styling.
+3. [**Use layouts to build reusable site sections.**](/tutorial/part-three/) Using layouts makes the site-building less repetitive. Using self-contained components allows us to reuse them in different places on a site (like headers, footers, sidebars, etc). In this section, we’ll look deeper into what layouts are, and how we use them.
 
```

Comment:
I completely forgot to mention this earlier; let's use "you" in place of "we" or "us" (and "your" instead of "our," etc.) whenever possible and only use "we" when "you" doesn't make sense. So all these will read like "you'll start by installing all the tools you need to begin development on a Gatsby site..."

Reply:
I can actually do this quickly once all other edits are made before I merge everything.

Reply:
As I'm going through everything, I'm updating this -- definitely note/edit where you see, if I've missed, please.